app/controllers/PackagedContractPackerController.scala

```diff
@@ -44,27 +46,27 @@ class PackagedContractPackerController @Inject()(
   val form = formProvider()
 
 
-  def onPageLoad(mode: Mode): Action[AnyContent] = (identify andThen getData) {
+  def onPageLoad(mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData andThen checkSub) {
```

Comment:
Again, why was requireData missing? Was this deliberate?

Reply:
I think this is the same as the other page